bevy_lint/src/lints/restriction/missing_reflect.rs

```diff
@@ -221,34 +211,32 @@ struct TraitType {
 }
 
 impl TraitType {
-    fn from_local_crate<'tcx>(
-        tcx: TyCtxt<'tcx>,
-        trait_path: &[&str],
-    ) -> impl Iterator<Item = Self> + use<'tcx> {
+    fn from_local_crate<'tcx, 'a>(
+        cx: &'a LateContext<'tcx>,
```

Comment:
`PathLookup::get()` requires a `LateContext`, not just a `TyCtxt`, which is why it's changed here.